packages/react-core/src/components/DescriptionList/DescriptionListText.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import styles from '@patternfly/react-styles/css/components/DescriptionList/description-list';
+import { css } from '@patternfly/react-styles';
+
+export interface DescriptionListTextProps extends React.HTMLProps<HTMLDivElement> {
+  /** Anything that can be rendered inside of list text */
+  children: React.ReactNode;
```

Comment:
I think we just need the children prop here.  Then the consumer can place whatever the want in the `<dt>`.  I don't see any related classes to any sub elements  of the `<dt>` in core.